pywb/templates/search.html

```diff
@@ -67,120 +68,126 @@ <h4 class="display-4">
+                                digits{% endtrans %} </div>
                         </div>
                     </div>
-                </div>
-                <div class="col-6">
-                    <label class="sr-only" for="dt-to" aria-label="Date/Time Range To">{% trans %}To:{% endtrans %}</label>
-                    <div class="input-group">
-                        <div class="input-group-prepend">
-                            <div class="input-group-text">{% trans %}To:{% endtrans %}</div>
-                        </div>
-                        <input id="dt-to" type="number" name="date-range-to" class="form-control" pattern="^\d{4,14}$">
-                        <div class="invalid-feedback" id="dt-to-bad">
-                            {% trans %}Please enter a valid <b>To</b> timestamp. Timestamps may be 4 <= ts <=14 digits{% endtrans %}
-                        </div>
-                    </div>
-                </div>
-            </div>
-            <div class="form-group mt-3">
-                <div class="form-row">
                     <div class="col-6">
-                        <p>{% trans %}Filtering{% endtrans %}</p>
-                    </div>
-                    <div class="col-6">
-                        <button id="clear-filters" class="btn btn-outline-warning float-right" type="button">
-                            {% trans %}Clear Filters{% endtrans %}
-                        </button>
-                        <button id="add-filter" class="btn btn-outline-secondary float-right mr-2" type="button">
-                            {% trans %}Add Filter{% endtrans %}
-                        </button>
-                    </div>
-                </div>
-                <div class="form-row">
-                    <div class="col-6">
-                        <div class="row pb-1">
-                            <label for="filter-by" class="col-form-label col-3">{% trans %}By:{% endtrans %}</label>
-                            <select id="filter-by" class="form-control col-7">
-                                <option value="" selected></option>
-                                <option value="mime">{% trans %}Mime Type{% endtrans %}</option>
-                                <option value="status">{% trans %}Status{% endtrans %}</option>
-                                <option value="url">{% trans %}URL{% endtrans %}</option>
-                            </select>
+                        <label class="sr-only" for="dt-to" aria-label="Date/Time Range To">{% trans %}To:{% endtrans
+                            %}</label>
+                        <div class="input-group">
+                            <div class="input-group-prepend">
+                                <div class="input-group-text">{% trans %}To:{% endtrans %}</div>
+                            </div>
+                            <input id="dt-to" type="number" name="date-range-to" class="form-control"
+                                pattern="^\d{4,14}$">
```

Comment:
There's a bunch of line break changes in this commit that don't seem related but they also look harmless. I assume this is just your editor keeping lines from getting too long? It's NBD.